sessions/R-estimation-and-the-renewal-equation.qmd

```diff
@@ -234,25 +235,18 @@ data <- list(
   ip_pmf = ip_pmf
 )
 r_inf_fit <- mod$sample(
-  data = data, refresh = 0, show_exceptions = FALSE, show_messages = FALSE
+  data = data, refresh = 0, show_exceptions = FALSE, show_messages = FALSE, parallel_chains = 4
```

Comment:
can we expect everyone will have >4 cores?

Reply:
this is just running each chain in parallel right? i.e max of 4 cores? but could drop it down to 2. I think if the models are working better probably don't need at all